pages/about.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Super SportCar Global Race | About</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="/Assets/sass/about.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"/>
+</head>
+<body class="scroll bg-light" id="body">
+  <div class="container-fluid">
+    <header class="row">
+      <div class="social">
+        <ul>
+          <li class="social-icon"><a href="#">Facebook <i class="fab fa-facebook"></i></a></li>
+          <li class="social-icon"><a href="#">Instagram <i class="fab fa-instagram"></i></a></li>
+          <li class="social-icon"><a href="#">Twitter <i class="fab fa-twitter"></i></a></li>
+        </ul>
+      </div>
+      <nav class="navbar navbar-expand-lg navbar-light">
+          <a class="navbar-brand" href="/index.html"><i class="fas fa-car"></i></a>
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="../index.html">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">About</a>
+              </li>
+            </ul>
+          </div>
+      </nav>
+      <div class="col-12">
```

Comment:
- [ ] **The hero section of your about page does not match the original design. Please fix it.**

Original Design|Cloned Design 
:-------------------:|:------------------:
![image](https://user-images.githubusercontent.com/59168430/124211656-1cd86900-db0b-11eb-9e0b-968616424385.png)|![image](https://user-images.githubusercontent.com/59168430/124211722-3d082800-db0b-11eb-8c4a-5c027c97ff60.png)

- The font hierarchy does not match the original design. For instance, the block text has a lower hierarchy than the text "Please contact us....."
- The call to action is missing in the cloned version. Please add it**